### PR TITLE
Implement a global "expand/collapse elements" button

### LINKED ISSUE #6025: Usability: Add new button for toggling all content elements at once
### Description

image

This strongly improves the ability to find an element in a list when they would otherwise be hard to distinguish. As of now, in such a case, the user would have to closely look at every element and expand every similar-looking one to find what they are looking for.

Furthermore, this makes it possible to quickly visually scan the list for something specific that is only visible in expanded state. Similarly, when using Ctrl+F to search for a word on the page, the context is not visible, which would also be solved with this improvement.

### PR DESCRIPTION
**Pull request by aschempp**
Implements https://github.com/contao/contao/issues/6025

~What I have not figured out yet (and could not find any useful resources on the internet) was how to _efficiently_ localize the controller. Passing 6 labels on ever instantiation feels wrong to me, but having a global array feels wrong too 🤷 any ideas?~

### CODE REVIEW

**Comment by fritzmg** [comment]
I'd just output the labels as JSON in an additional `data` attribute.

**Comment by aschempp** [comment]
> I'd just output the labels as JSON in an additional `data` attribute.

but this would output the labels **for every `data-controller`** again and again?

**Comment by fritzmg** [comment]
Which is good enough, since there will likely only be one toggle button per back end response, or won't there?

**Comment by aschempp** [comment]
no, the controller is on every content element that implements the dynamic height property. It dynamically generates the global toggle operation on the fly.

**Comment by aschempp** [comment]
I have rewritten the whole controller script and approach. Instead of setting a `limit_height` (and corresponding height class) in your custom row callback, you can now set a `list.sorting.limitHeight` property on the DCA, and it will automatically generate the necessary operation and wrapper DIVs in DC_Table 😎

File: core-bundle/contao/themes/flexible/styles/main.css [hunk 1]
```diff
@@ -1113,6 +1113,7 @@ table.tl_listing {
 }
 
 .tl_file, .tl_file_list {
+	position: relative;
```
Comment (leofeyer) [inline core-bundle/contao/themes/flexible/styles/main.css:1116]: This change seems unrelated. What does it do?

**Review by leofeyer — APPROVED** [review]

File: core-bundle/contao/themes/flexible/styles/main.css (hunk 1 above)
Comment (aschempp) [reply core-bundle/contao/themes/flexible/styles/main.css:1116]: it is not unrelated. It adds relative positioning for comments (mode 1) to fix the expand/collapse button. Try removing it in the comments list when this PR is applied (in the Contao demo).

**Comment by leofeyer** [comment]
Thank you @aschempp.
